### PR TITLE
Update omnisharp extensions 0 19 7

### PR DESCRIPTION
**Pull request by bryankenote**
resolves https://github.com/FacilityApi/FacilityLanguageServer/issues/7

### CODE REVIEW

File: src/Facility.LanguageServer/FsdHoverHandler.cs [hunk 1]
```diff
@@ -26,9 +27,9 @@ public void SetCapability(HoverCapability capability)
 		{
 		}
 
-		public async Task<Hover> Handle(TextDocumentPositionParams request, CancellationToken token)
+		public async Task<Hover> Handle(HoverParams request, CancellationToken cancellationToken)
 		{
-			Uri documentUri = request.TextDocument.Uri;
+			DocumentUri documentUri = request.TextDocument.Uri;
```
Comment (ddunkin) [inline src/Facility.LanguageServer/FsdHoverHandler.cs:32]: I'd just go with `var` here (and everywhere).

File: src/Facility.LanguageServer/FsdRequestHandler.cs [hunk 2]
```diff
@@ -27,9 +28,9 @@ protected void SetService(Uri documentUri, ServiceInfo service)
 			m_services[documentUri] = service;
 		}
 
-		protected ServiceInfo GetService(Uri documentUri)
+		protected ServiceInfo GetService(DocumentUri documentUri)
 		{
-			return m_services.TryGetValue(documentUri, out ServiceInfo service) ? service : null;
+			return m_services.TryGetValue(documentUri.ToUri(), out ServiceInfo service) ? service : null;
```
Comment (ddunkin) [inline src/Facility.LanguageServer/FsdRequestHandler.cs:33]: If `DocumentUri` would be a valid key, change the dictionary to use that. `SetService` should use the same type, i.e. the API should be consistent.

File: src/Facility.LanguageServer/Program.cs [hunk 3]
```diff
@@ -1,18 +1,18 @@
-using Facility.Definition;
 using Facility.LanguageServer;
-using Microsoft.Extensions.Logging;
```
Comment (ddunkin) [inline src/Facility.LanguageServer/Program.cs:3]: Do we still need the `Microsoft.Extensions.Logging` package reference?

**Comment by bryankenote** [comment]
I referred to the current example for the changes to the program.cs
https://github.com/OmniSharp/csharp-language-server-protocol/blob/05e543600b439dc3fb20b877f099e685cd808985/sample/SampleServer/Program.cs#L40

File: src/Facility.LanguageServer/Program.cs (hunk 3 above)
Comment (bryankenote) [reply src/Facility.LanguageServer/Program.cs:3]: This builds but there is no logging. The current example uses Serilog on the Omnisharp repo. We could just copy that.

Comment (ddunkin) [reply src/Facility.LanguageServer/Program.cs:3]: Where do the logs go? I'd be more inclined to use `Microsoft.Extensions.Logging.Console`.

Comment (ddunkin) [reply src/Facility.LanguageServer/Program.cs:3]: OTOH, I don't think logs went anywhere before, and I don't know that there's anything being logged we'd care to see, so we could leave it unconfigured.

Comment (bryankenote) [reply src/Facility.LanguageServer/Program.cs:3]: right it didn't see like we were keeping track of that. I added https://github.com/FacilityApi/FacilityLanguageServer/pull/9/commits/2be83828e896a28fc41f0fc103c12f8217a66ebb as an example. Might need further configuration to be practical. But happy to revert or improve this.

**Review by ddunkin — APPROVED** [review]
🎉

**Pull request merged**